__tests__/users.ts

```diff
@@ -6991,3 +7123,53 @@ describe('mutation uploadResume', () => {
     expect(body.errors[0].message).toEqual('File type not supported');
   });
 });
+describe('mutation updateNotificationSettings', () => {
+  const MUTATION = `mutation UpdateNotificationSettings($notificationFlags: JSON!) {
+    updateNotificationSettings(notificationFlags: $notificationFlags) {
+      _
+    }
+  }`;
+
+  it('should update notification settings', async () => {
```

Comment:
```suggestion
  it('should overwrite notification settings during update', async () => {
```

just so its clear it is supposed to happen as you defined in RFC